language/move-prover/doc/user/spec-lang.md

```diff
@@ -212,6 +213,63 @@ spec module {
 Is seen in the example, they can be generic. Helper functions can also access global state as done in the
 example.
 
+## Assume and Assert Conditions
```

Comment:
I moved the section about in-body spec blocks before function-level spec blocks because my explanation for modifies was using assertions.